css/src/elementor.css

```diff
@@ -74,6 +79,89 @@
 	background: none;
 }
 
+.yoast-gutenberg-modal input[type="radio"] {
+	appearance: none;
+	border-radius: 50%;
+	border: 2px solid #6c7781;
+	margin-right: 12px;
+	transition: none;
+	box-shadow: 0 0 0 transparent;
+	height: 1rem;
+	padding: 0;
+	vertical-align: middle;
+	width: 1rem;
+	min-width: 1rem;
+}
+
+.yoast-gutenberg-modal input[type="radio"]:checked {
+	background: #11a0d2;
+	border-color: #11a0d2;
+}
+
+input[type="radio"]:checked::before {
+	content: "";
+	float: left;
+	vertical-align: middle;
+	border-radius: 50%;
+	line-height: 1.14285714;
+	width: 6px;
+	height: 6px;
+	margin: 3px 0 0 3px;
+	background-color: #fff;
+}
+
+/* Upsell modal close-button tooltips */
+.yoast-gutenberg-modal {
+	overflow: visible;
+}
+
+.yoast-gutenberg-modal .components-modal__header .components-button {
+	overflow: visible;
+}
+
+.yoast-gutenberg-modal .components-modal__header .components-button > span {
+	width: 0;
+	height: 0;
+}
+
+.yoast-gutenberg-modal .components-modal__header .components-button .components-tooltip {
+	position: absolute;
+	top: 0 !important;
+	left: 20px !important;
+}
+
+.yoast-gutenberg-modal .components-modal__header .components-tooltip::before, .yoast-gutenberg-modal .components-modal__header .components-tooltip::after {
+	top: 0;
+}
+
+/* Tab-focus styling */
+.yoast div:focus {
+	outline: 0;
+}
+
+.yoast a:focus, .yoast .button-link:focus {
+	color: #124964;
+	box-shadow: 0 0 0 1px #5b9dd9, 0 0 2px 1px rgba(30, 140, 190, 0.8);
+	outline: 1px solid transparent;
+}
+
+.yoast a.dashicons:focus {
+	color: #1e8cbe;
+}
+
+.yoast input[type="radio"]:checked:focus {
```

Comment:
I cannot find the equivalent of this styling in the Gutenberg sidebar. Did you intend to add a box shadow on focus to the radio buttons?

Reply:
This mimics Gutenberg behaviour:
<img width="1446" alt="Screenshot 2020-11-10 at 10 28 54" src="https://user-images.githubusercontent.com/69148430/98655593-ee213e80-233f-11eb-9151-15d9e04b51bb.png">